spec/rspec/rails/matchers/action_cable/have_broadcasted_to_spec.rb

```diff
@@ -226,5 +226,43 @@ def broadcast(stream, msg)
         end
       end
     end
+
+    it "has a description when not chained" do
```

Comment:
“Chained” can mean two things: adding a qualifier like from_message, and compounding with and/or. In this context, when you say that it is not chained, it is unclear what is meant.